docs_user/modules/proc_adopting-image-service-with-block-storage-backend.adoc

```diff
@@ -77,8 +75,9 @@ spec:
             - storage
```

Comment:
I have two questions on step 1:

1. Does the template also need “secret:” ?
2. (Hackfest feedback) Does the indentation look correct for “spec: type: LoadBalancer”?